source/_docs/configuration/templating.markdown

```diff
@@ -916,7 +916,6 @@ The numeric functions and filters raise an error if the input is not a valid num
   - `round(1, "half", default)` will always round to the nearest .5 value. `precision` should be 1 for this mode
 - Filter `value_one|bitwise_and(value_two)` perform a bitwise and(&) operation with two values.
 - Filter `value_one|bitwise_or(value_two)` perform a bitwise or(\|) operation with two values.
-- Filter `value_one|bitwise_xor(value_two)` perform a bitwise xor(\^) operation with two values.
```

Comment:
Why is this removed?

Reply:
That was unintended. How can I best add it back?